pkgs/development/libraries/linenoise/default.nix

```diff
@@ -0,0 +1,29 @@
+  { stdenv, fetchgit }:
+
+  stdenv.mkDerivation rec {
+    name = "linenoise-${version}";
+    version = "git-2016-09-30";
+
+    src = fetchgit {
+      url = "https://github.com/antirez/linenoise.git";
+      rev = "c894b9e59f02203dbe4e2be657572cf88c4230c3";
+      sha256 = "0wasql7ph5g473zxhc2z47z3pjp42q0dsn4gpijwzbxawid71b4w";
+    };
+
+    phases = ["unpackPhase" "buildPhase" "installPhase"];
```

Comment:
Oh, I didn't know about the noop.
So you prefer that I explicitly says that each phases are noop rather than saying that only those phases are used?